lib/drivers/insuletDriver.js

```diff
@@ -896,6 +896,13 @@ module.exports = function (config) {
             .with_status(postsuspend)
             .done();
           break;
+        // for alarm codes not documented in the spec
+        default:
+          if (postalarm) {
+            postalarm = postalarm.with_alarmType('other')
```

Comment:
Yes, because these alarm codes aren't documented in the spec, we don't want to expose any info we shouldn't in the payload. They will just be "other" alarms until when/if we get more info in the spec we've been given to implement.